plugins/modules/activation.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+
+# (c) 2023, Nikhil Jain <nikjain@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: activation
+author: "Nikhil Jain (@jainnikhil30)"
+short_description: Create, restart or delete rulebook activations in EDA Controller.
```

Comment:
Other than create and delete, no other operation such as restart, stop, disable, enable an existing activation is supported in this PR.

Reply:
That is correct changed the description